### ISSUE TITLE
density doesn't work stably on smaller screen sizes

### ISSUE THREAD

**Issue by WebDeg-Brian** [Wufe/react-particles-js#60]
### Source code:

I probably have done something wrong. Here is the my particles setup:
```
import React from 'react';

import styled from 'styled-components';

import ParticlesCanvas from 'react-particles-js';

const StyledParticlesWrapper = styled.div`
  position: absolute;
  width: 100%;
  height: 100%;
  overflow: hidden;
`;

export default Particles = props => {
  return (
    <StyledParticlesWrapper>
      <ParticlesCanvas params={{
        particles: {
          number: {
            value: 80,
            density: {
              enable: true,
              value_area: 800
            }
          }
        }
      }} />    
    </StyledParticlesWrapper>
  );
}
```

### Behaviour

When the page is loaded on a big screen size, `density` works. When resizing to a small enough screen size, `density` sometimes works, sometimes doesn't. I tested the app on iPhoneX and Galaxy S9 screen size as well as responsive mode.

Particles when loaded on a big screen size:
![image](https://user-images.githubusercontent.com/43169879/47936145-b1aae480-ded4-11e8-9004-f6b61ef64ed0.png)

Particles when being resized (Sometimes it works):
![image](https://user-images.githubusercontent.com/43169879/47936936-2e3ec280-ded7-11e8-967a-9d6104ef41c6.png)

Particles when being resized (Sometimes it doesn't work):
![image](https://user-images.githubusercontent.com/43169879/47937119-a907dd80-ded7-11e8-96b1-db232cf60865.png)

How should I fix it? Thanks in advance

My project repo: [Eagle Nest](https://www.github.com/WebDeg-Brian/eagle-nest)
The particles file: [Particles.js](https://github.com/WebDeg-Brian/eagle-nest/blob/master/imports/ui/components/HeroBanner/Particles.js)

**Comment by WebDeg-Brian** [comment]
I also tried this code and notice that the possibilities of the `density` failure decreases. But still, it happens
```
import React from 'react';

import styled from 'styled-components';

import ParticlesCanvas from 'react-particles-js';

const StyledParticlesWrapper = styled(ParticlesCanvas)`
  position: absolute;
  width: 100%;
  height: 100%;
  overflow: hidden;
`;

export default Particles = props => {
  return (
    <StyledParticlesWrapper params={{
        particles: {
          number: {
            value: 80,
            density: {
              enable: true,
              value_area: 800
            }
          }
        }
      }} >    
    </StyledParticlesWrapper>
  );
}
```

**Comment by Wufe** [comment]
There was a bug creating a concurrency of events. This bug has been resolved and will be included in next release (v2.4.0)
